src/here.ts

```diff
@@ -31,7 +31,7 @@ const program = require('commander');
 const settings = require('user-settings').file('.herecli');
 const latestVersion = require('latest-version');
 
-const commands = ["xyz", "xs","c","configure", "transform","tf", "help", "geocode","gc"];
+const commands = ["xyz", "studio","xs","c","configure", "transform","tf", "help", "geocode","gc"];
```

Comment:
please add alias 's' as well, also think about the name as just s is confusing

Reply:
Have added ```s``` parameter in commands alias. Can't think of any other alias names for now. thought of ```std``` or ```stdo``` but ```s``` seems fine. Please suggest if you think any other alias names would be fine here